### PR TITLE
Custom events iOS SDK

### PR DESCRIPTION
**Pull request by jsani-radar**
List out custom events iOS functions explicitly

### CODE REVIEW

File: docs/sdk/ios.mdx [hunk 1]
```diff
@@ -1462,3 +1462,42 @@ Radar.getMatrix(
...
+
+```swift
+Radar.sendEvent(
+    name: name,
+    metadata: metadata,
+) { (status: RadarStatus, location: CLLocation?, events: [RadarEvent]?, user: RadarUser?) in
+  // do something with location, events, user
+  // events[0] is the custom event
+}
+```
```
Comment (jrtibbetts) [inline docs/sdk/ios.mdx:1488]: I'm itching' to add `async` Swift examples to all the docs, too.

**Review by jrtibbetts — APPROVED** [review]

File: docs/sdk/ios.mdx [hunk 2]
```diff
@@ -1462,3 +1462,42 @@ Radar.getMatrix(
 
   </TabItem>
 </Tabs>
+
+### Custom events
+
+With the [custom events API](/api#create-a-custom-event), create a custom event, such as a conversion or purchase, to analyze alongisde your app's location activity:
```
Comment (corypisano) [inline docs/sdk/ios.mdx:1468]: ah - alongside typo

Comment (jsani-radar) [reply docs/sdk/ios.mdx]: 👏🏽

File: docs/sdk/ios.mdx (hunk 1 above)
Comment (jsani-radar) [reply docs/sdk/ios.mdx:1488]: open a PR! ✅

**Pull request merged**